presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py

```diff
@@ -0,0 +1,62 @@
+from PIL import Image, ImageChops
+from presidio_image_redactor.image_analyzer_engine import ImageAnalyzerEngine
+import matplotlib
+import io
+from matplotlib import pyplot as plt
```

Comment:
Is matplotlib a necessity here? We'd rather not add additional dependencies unless mandatory. Can we use PIL for this logic?

Reply:
Hi, I use ax.annotate to draw entity_type above the empty box of the words, then transfer fig to image.
      fig, ax = plt.subplots()
      rect = matplotlib.patches.Rectangle(
                    (x0, y0), x1 - x0, y1 - y0, edgecolor="b", facecolor="none"
                )
      ax.annotate(
                    entity_type,
                    xy=(x0 - 3, y0 - 3),
                    xycoords="data",
                    bbox=dict(boxstyle="round4,pad=.5", fc="0.9"),
                )
    
       im_from_fig = fig2img(fig)

Reply:
@zhangabner in this case could you please add matplotlib to setup.py and Pipfile?

Reply:
@omri374 sure, I added matplotlib to setup.py and Pipfile.

Reply:
@zhangabner I see there's some merge conflicts. Could you please resolve them?

Reply:
@navalev , resolved.